traitsui/qt4/ui_panel.py

```diff
@@ -257,7 +257,6 @@ def panel(ui):
         panel = QtGui.QTabWidget()
         # Identify ourselves as being a Tabbed group so we can later
         # distinguish this from other QTabWidgets.
-        panel.setProperty("traits_tabbed_group", True)
```

Comment:
Yes, it is indeed unrelated. I read somewhere that pyside had issues (segfault) when exiting with dynamic properties, so it was one of my attempts to fix the issue.
It is dead code btw, so i thought i might as well remove it.
I can separate it out in a different PR if you think so. Same for old-style->new-style signal connections